core/app_main.c

```diff
@@ -21,6 +21,8 @@
 #include "esp/spi_regs.h"
 #include "esp/dport_regs.h"
 #include "esp/wdev_regs.h"
+#include "esp/wdt_regs.h"
+#include "esp/rtcmem_regs.h"
```

Comment:
the `RTCMEM_BACKUP` define inside `rtcmem_regs.h`, `WDT` inside `wdt_regs.h`. It's normal to me, except that I compiled together with esphttpd.